src/demo/share/jfc/SwingSet2/resources/swingset_ja.properties

```diff
@@ -202,24 +202,24 @@ ButtonDemo.imagetogglebuttons=\u30A4\u30E1\u30FC\u30B8\u30FB\u30C8\u30B0\u30EB\u
 ButtonDemo.textcheckboxes=\u30C6\u30AD\u30B9\u30C8\u30FB\u30C1\u30A7\u30C3\u30AF\u30FB\u30DC\u30C3\u30AF\u30B9
 ButtonDemo.imagecheckboxes=\u30A4\u30E1\u30FC\u30B8\u30FB\u30C1\u30A7\u30C3\u30AF\u30FB\u30DC\u30C3\u30AF\u30B9
 
-ButtonDemo.button1=1 
+ButtonDemo.button1=1\u0020
 ButtonDemo.button2=2
 ButtonDemo.button3=<html><font size=2 color=red><bold>3</font></html>
 
-ButtonDemo.radio1=\u30DC\u30BF\u30F31 
+ButtonDemo.radio1=\u30DC\u30BF\u30F31\u0020
 ButtonDemo.radio2=\u30DC\u30BF\u30F32
 ButtonDemo.radio3=\u30DC\u30BF\u30F33
 ButtonDemo.radioX=<html><font size=2 color=red>3<bold>(HTML)</bold></font></html>
 
-ButtonDemo.check1=1 
+ButtonDemo.check1=1\u0020
 ButtonDemo.check2=2
 ButtonDemo.check3=3
 ButtonDemo.checkX=<html><font size=2 color=red>3<bold>(HTML)</bold></font></html>
 
 ButtonDemo.customradio=\u30AB\u30B9\u30BF\u30E0\u306E"\u30E1\u30BF\u30EA\u30C3\u30AF\u8ABF"\u30E9\u30B8\u30AA\u30FB\u30DC\u30BF\u30F3
 ButtonDemo.customcheck=\u30AB\u30B9\u30BF\u30E0\u306E\u96FB\u7403\u30C1\u30A7\u30C3\u30AF\u30FB\u30DC\u30C3\u30AF\u30B9
 
-ButtonDemo.phone=\u96FB\u8A71 
+ButtonDemo.phone=\u96FB\u8A71\u0020
 ButtonDemo.write=\u66F8\u8FBC\u307F
 ButtonDemo.peace=\u30D4\u30FC\u30B9
```

Comment:
It's only here that the space is present, while `write` and `peace` don't, nor does the en locale. It goes into the alt text of an icon, so a space afterwards is unnecessary.